MAUI/Scheduler/drag-and-drop-appointments.md

```diff
@@ -0,0 +1,191 @@
+{
+    var appointment = e.Appointment;
+    e.Cancel = false;
+}
+{% endhighlight %}
+{% endtabs %}
+
+#### Disabling dragging when the appointment is AllDay appointment
+You can enable/disable the appointment dragging based on the requirement by using `Cancel` property in the `AppointmentDragStartingEventArgs` argument of Scheduler `AppointmentDragStarting` event.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDragStarting += OnSchedulerAppointmentDragStarting;
+
+...
+
+private void OnSchedulerAppointmentDragStarting(object? sender, AppointmentDragStartingEventArgs e)
+{
+    var appointment = e.Appointment as SchedulerAppointment;
+    if (appointment.IsAllDay)
+    {
+        e.Cancel = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+Here, appointment dragging is disabled when the appointment is `AllDay` appointment.
+
+## Get the dragging appointment position and time
+you can get the dragging appointment details, position and time of the particular location by using `AppointmentDragOver` event. This event will be continuously triggered when the appointment is being dragged. The `AppointmentDragOverEventArgs` argument contains the following properties.
+
+`Appointment` - Gets the appointment that is being dragged.
+`DragPoint`- Gets the dragging point of the scheduler appointment user interface.
+`DragTime` - Gets the dragging time of the appointment object being dragged.
+`DragResource` - Gets the original resource of the appointment that is being dragged.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDragOver += OnSchedulerAppointmentDragOver;
+
+...
+
+private void OnSchedulerAppointmentDragOver(object? sender, AppointmentDragOverEventArgs e)
+{
+    var appointment = e.Appointment;
+    var draggingPoint = e.DragPoint;
+    var draggingTime = e.DragTime;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Handle appointment dropping
+Using `AppointmentDrop` event you can get the dropping appointment details, position, time and you can handle whether the appointment can be dropped to the specific position or not. This event will trigger after dropping the appointment. The `AppointmentDropEventArgs` argument contains the following properties
+
+`Appointment` - Gets the appointment that is being dragged and dropped.
+`DropTime`- Gets or sets the date and time at which the appointment is being dropped.
+`SourceResource` - Gets the original resource of the appointment that is being dropped.
+`TargetResource` - Gets the resource into which the appointment is being dropped.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDrop += OnSchedulerAppointmentDrop;
+
+...
+
+private void OnSchedulerAppointmentDrop(object? sender, AppointmentDropEventArgs e)
+{
+        var appointment = e.Appointment;
+        e.Cancel = false;
+        var dropTime = e.DropTime;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the Drag and Drop environment
```

Comment:
## Change the Drag and Drop settings
You can change the appointment of drag and drop settings in Scheduler by using `DragDropSettings` property.

Reply:
changed title